templates/profile/kubernetes/haproxy/services.d/api.cfg.erb

```diff
@@ -7,5 +7,6 @@ frontend kubernetes-api-front
 
 backend kubernetes-api-back
   mode tcp
-  option tcp-check
   balance roundrobin
+  option httpchk HEAD /
+  http-check expect status 403
```

Comment:
I'm not especially happy running essentially `curl -k` and considering `403 Forbidden` to be "success," but the aim here is not to test our PKI but rather to check that a server is ready to receive requests, and if not, remove it from load balancing. Assuming that we **don't** have valid certs, 403 is in fact the expected behavior.

If some later version of kubernetes decides to switch to `401` or something, I believe it will be impossible not to notice in the test cluster.